src/api/app/controllers/webui/request_controller.rb

```diff
@@ -49,7 +49,27 @@ def show
       end
 
       # TODO: Remove this `render` line once request_show_redesign is rolled out
-      render :beta_show
+      respond_to do |format|
+        format.html do
+          render 'beta_show', locals: {
```

Comment:
Do you see anyone else using render+locals in a controller in this app? I'm not quite sure this adds clarity...

Reply:
It goes against the Rails convention. Top-level views always use instance variables, then they pass those as local variables to partials.